### PR TITLE
build shorter, more readable urls for capi images

### PR DESCRIPTION
**Pull request by wheresrhys**

### CODE REVIEW

**Comment by i-like-robots** [comment]
Is this still a thing?

File: src/helpers/build-image-service-url.js [hunk 1]
```diff
@@ -1,12 +1,15 @@
 import qs from 'querystring';
 
+const capiRX = /^http:\/\/com\.ft\.imagepublish\.prod\.s3\.amazonaws\.com\/([0-9a-f]{8}-[0-9a-f]{4}-[0-9a-f]{4}-[0-9a-f]{4}-[0-9a-f]{12})/
```
Comment (JakeChampion) [inline src/helpers/build-image-service-url.js]: The regex will need to be `.prod(-us)?` in place of `.prod` as there are two s3 buckets, one in eu and one in us.

Comment (JakeChampion) [inline src/helpers/build-image-service-url.js]: We just found this ourselves with the new image service, here is the PR we made -- https://github.com/Financial-Times/origami-image-service/pull/87

**Comment by wheresrhys** [comment]
intention is for it to be a thing. I'm slowly working back through all my open PRs

**Comment by wheresrhys** [comment]
:+1:

**Pull request merged**